packages/shared/src/components/comments/MainComment.spec.tsx

```diff
@@ -20,6 +21,7 @@ const renderLayout = (
   user: LoggedUser = null,
 ): RenderResult => {
   const defaultProps: MainCommentProps = {
+    post,
```

Comment:
Noticed tests were failing because `post` was missing. `post` prop is required by `MainCommentProps` so I think this was just a test error.